spec/controllers/health_controller_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "spec_helper"
+
+describe HealthController, type: :controller do
+  describe "GET #ping" do
+    it "returns a successful response with status ok" do
+      get :ping
+      expect(response.status).to eq(200)
+      expect(JSON.parse(response.body)).to eq({"status" => "ok"})
```

Comment:
Does this work?

```suggestion
      expect(response.parsed_body).to eq({"status" => "ok"})
```

Reply:
sure does, forgot about that !

Reply:
I should get this diff tattooed on my forehead 😝